Adds optional defaults for php-fpm user & group.

entendu opened the pull request:
Fixes #781 which is solved upstream but not yet referenced in requirements. Also adds an optional user/group param for php-fpm.

geerlingguy commented on default.config.yml:
```diff
@@ -230,6 +230,11 @@ php_max_input_vars: "4000"
 php_enable_php_fpm: true
 php_fpm_listen: "127.0.0.1:9000"
 
+# If you want to run php-fpm as a user/group other than your OS's default user,
```
Instead of adding in this file, can you just add a note on this docs page (http://docs.drupalvm.com/en/latest/other/webservers/#using-nginx) mentioning these two variables and how to set them if switching to `nginx`? Since it's a smaller use case (at least at this point in time), I'd rather not add the four lines to `default.config.yml`.

geerlingguy commented on provisioning/requirements.yml:
```diff
@@ -36,7 +36,7 @@
   version: 3.2.0
 - src: geerlingguy.php
-  version: 3.2.1
+  version: 3.2.2
```
Good catch! I thought I had updated the role in requirements... but apparently not.

entendu commented:
@geerlingguy, good call. Added some explanation to webservers.md (http://docs.drupalvm.com/en/latest/other/webservers/#using-nginx) and reverted changes to default.config.yml.
